x/accounts/keeper.go

```diff
@@ -416,6 +417,17 @@ func (k Keeper) maybeSendFunds(ctx context.Context, from, to []byte, amt sdk.Coi
 	return nil
 }
 
+func (k Keeper) isAccountTypeRegistered(ctx context.Context, accountType string) (bool, error) {
+    _, err := k.AccountsByType.Get(ctx, []byte(accountType))
+    if err != nil {
+        if strings.Contains(err.Error(), "not found") {
```

Comment:
We can check the typed error collections.ErrNotFound

Reply:
```
func (k Keeper) isAccountTypeRegistered(ctx context.Context, accountType string) bool {
    _, err := k.AccountsByType.Get(ctx, []byte(accountType))
    if err != nil && errors.Is(err, collections.ErrNotFound) {
        return false
    }
    return true
}
```
Would it be okay to make the modification like this?